wordpress-mobile/AztecEditor-iOS
# Add tests for testing multiple figures in sequence.

This PR at the moment is just a starting point to try and track the issue with image with captions on a sequence not being parsed correctly.

The issue looks to be in the serialization of the `figure` element back to HTML not in the caption shortcode processor.

To test:
 - Run the unit tests
 - See the `testMultipleFigureCaptionAreProperlyParsed()` test fail
